amundsen_application/static/js/components/common/Badges/styles.scss

```diff
@@ -1,10 +1,51 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
-
 @import 'variables';
-
 .clickable-badge {
   &:hover {
     cursor: pointer;
   }
+  .label {
+    padding: 0 0;
+  }
+
+  [class^="badge-overlay-"] { 
+    font-size: $font-size-base;
+    text-align: center;
+    padding: .2em .6em .3em;
+
+    &:hover, &:focus {
+      color: $badge-text-color;
+      height: $badge-height;
+      width: 100%;
+      border-radius: 20px;
+    }
+
+    &:focus {
```

Comment:
tldr; I think this is an issue we can come back to

In practice the focus effect applies cases when components are keyboard accessible, i.e. being able to use the `Tab` key to navigate to this component and the `Enter` key when it's focused to trigger the action. However the way this component is written it is currently not keyboard accessible. If you try it out and you'll see you won't be able to reach this component hitting the `Tab` key.  And let's say it were accessible as written, I think the focus will actually end up on the parent element as opposed to the overlay, so then this css wouldn't apply regardless. 

More recently Eng has been trying to do a better job with writing accessible application code (but internally, this hasn't been a priority for the product), so what we can do is to have a TODO to update this component to be accessible. And then we would fix the focus state. Any thoughts or questions?

Reply:
I understand, I was using the focused option on inspect so I didn't try reaching it but I was wondering if the way the components were nester would be an issue. I found a potential solution just now that leverages children selection so when the .clickable-badge is focus we can change the style of the .badge-overlay-. I will give it a quick try see how that goes.

Reply:
Will create a separate PR to decouple Flag from clickable badge and make the component keyboard accessible.